### PR TITLE
Nutri analysis work

### PR DESCRIPTION
**Pull request by jennysiu**
Nutri analysis work
- html layout
- html classes & id
- render vitamins and minerals to nutri card
- moved nutri CSS to separate file temporarily

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -132,191 +145,214 @@ <h3>Looking for inspirations (Random recipes)</h3>
       </div>
     </section>
 
-    <!-- NUTRITIONAL ANALYSIS SECTION -->
-    <section class="nutritional-analysis-section">
+    <!-- *** NUTRITIONAL ANALYSIS SECTION *** -->
+    <section
+      class="section container-fluid"
```
Comment (willmowlam) [inline index.html]: Add class: d-none

File: assets/js/script.js [hunk 2]
```diff
@@ -1,18 +1,20 @@
-// Event listener for click id recipeNavigation and hide .nutritional-analysis-section
+// Event listener for click id recipeNavigation and hide #nutritional-analysis
 $("#recipeNavigation").on("click", function () {
   $(".recipe-search-section").removeClass("d-none");
   $(".randomRecipes").removeClass("d-none");
-  $(".nutritional-analysis-section").addClass("d-none");
+  $("#nutritional-analysis").addClass("d-none");
```
Comment (willmowlam) [inline assets/js/script.js]: Change ID to "#nutritional-analysis-container"

File: assets/js/script.js [hunk 3]
```diff
@@ -1,18 +1,20 @@
...
 });
 
 // Event listener for click id nutritionalNavigation and hide .recipe-search-section
 $("#nutritionalNavigation").on("click", function () {
   $(".recipe-search-section").addClass("d-none");
   $(".randomRecipes").addClass("d-none");
-  $(".nutritional-analysis-section").removeClass("d-none");
+  $("#nutritional-analysis").removeClass("d-none");
```
Comment (willmowlam) [inline assets/js/script.js]: "#nutritional-analysis-container"

File: assets/js/script.js [hunk 4]
```diff
@@ -1,18 +1,20 @@
...
 });
 
 // Hide .nutritional-analysis-section on load
 $(function () {
```
Comment (willmowlam) [inline assets/js/script.js]: Delete the entire doc ready function as the class is added to index.html to hide on load:

<section class="section container-fluid d-none" id="nutritional-analysis-container">

File: index.html (hunk 1 above)
Comment (jennysiu) [reply index.html]: @willmowlam doesn't the script.js add the d-none class to the section on load already?

**Review by willmowlam — APPROVED** [review]

**Pull request merged**